spec/requests/version_close_spec.rb

```diff
@@ -12,6 +12,6 @@
   it 'closes the version' do
     post "/objects/#{druid}/versionClose?version=3"
     expect(response).to be_successful
-    expect(WorkflowStep.where(druid: druid).count).to eq 13
+    expect(WorkflowStep.where(druid: druid).count).to eq 14
```

Comment:
seems like it would be easier to have a constant instead of updating this in so many places, but 🤷🏻‍♀️

Reply:
This happens so infrequently that I don't mind the extra 5 minutes.